base/src/org/compiere/acct/Doc.java

```diff
@@ -2527,14 +2526,15 @@ private String generateReverseWithOriginalAccounting() {
 	 * get Reversal Fact Accounts
 	 * @return
 	 */
-	private List<MFactAcct> getReversalFactAcct()
+	private List<MFactAcct> getReversalFactAcct(MAcctSchema asch)

```

Comment:
@carlosaparadam can please use complete variable description is best to developer a good DSL in code.

generateReverseWithOriginalAccounting(MAcctSchema originalAccountSchema) 

Also adding the comment parameter in documentation